packages/loll-app/index.js

```diff
@@ -8,17 +8,17 @@ function currentURL (loc = window.location) {
 }
 
 module.exports = function loll (r = []) {
-  let ref = null
   const router = createRouter(r)
 
   function mount (root, done) {
+    let element = root.firstElementChild
     const ev = mitt.default ? mitt.default() : mitt()
 
     function render (url = currentURL()) {
       ev.emit('navigate', url)
       return Promise.resolve(router.get(url))
         .then(next => requestAnimationFrame(() => {
-          patch(root, ref, (ref = next))
+          element = element ? patch(next, element) : root.appendChild(patch(next))
```

Comment:
@estrattonbailey Before it was necessary to keep track of a lot of stuff, but now it's just the element to update. To be honest, I would use the patch function directly, instead of creating a specialized render/mount function. But here's a way to do it.